src/fundus/publishers/de/boersenzeitung.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class BoersenZeitungParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("p")
+        _subheadline_selector = CSSSelector("h2.no-tts no-tts wp-block-ppi-subheading")
+        _topic_selector = CSSSelector("a[href^='/thema'] > span")
+        _paywall_selector = CSSSelector("storefront-html.paywall-headline > div")
+
+        @attribute
+        def title(self) -> Optional[str]:
+            fulltitle = self.precomputed.meta.get("og:title")
+            assert isinstance(fulltitle, str) #had to add this for mypy
+            title = str(fulltitle.split("|")[0].strip())
+            return title
```

Comment:
i think it would be easier doing this with a regex

```python
        _title_bloat_pattern = re.compile(r"\|.*")

        @attribute
        def title(self) -> Optional[str]:
            if fulltitle := self.precomputed.meta.get("og:title"):
                return re.sub(self._title_bloat_pattern, "", fulltitle).strip()
            return None
```